demisto_sdk/commands/common/constants.py

```diff
@@ -48,6 +49,75 @@
 XDRC_TEMPLATE_DIR = "XDRCTemplates"
 LAYOUT_RULES_DIR = "LayoutRules"
 
+FIRST_LEVEL_FOLDERS = {
```

Comment:
when we add new items we will need to add them to this list, which is hard to remember.
we have `demisto_sdk.commands.content_graph.common.ContentType.folders()` who doesn't need maintenance - consider using it.